samples/stock/interactive-docs/range-selector/demo.js

```diff
@@ -0,0 +1,70 @@
+(async () => {
+
+    // Load the dataset
+    const data = await fetch(
+        'https://demo-live-data.highcharts.com/aapl-ohlc.json'
+    ).then(response => response.json());
+
+    const chart = Highcharts.stockChart('container', {
+        series: [{
+            type: 'candlestick',
+            id: 'aapl-ohlc',
+            name: 'AAPL Stock Price',
+            data: data
+        }],
+        navigator: {
+            enabled: false
+        }
+    });
+
+    // Toggle range selector
+    const toggleButton = document.getElementById('toggle-range-selector');
+    toggleButton.addEventListener('click', () => {
+        const rangeSelectorEnabled = chart.rangeSelector.options.enabled;
+
+        chart.update({
+            rangeSelector: {
+                enabled: !rangeSelectorEnabled
+            }
+        });
+
+        toggleButton.innerText =
+        `${rangeSelectorEnabled ? 'Enable' : 'Disable'} range selector`;
+    });
+
+    // Update range selector buttons
+    const updateButton =
+        document.getElementById('update-range-selector-buttons');
+
+    updateButton.addEventListener('click', () => {
```

Comment:
To make the demo more interacitve maybe add some other options to make it togglable? nothing happens after the 1 click 🤔